.github/workflows/_func.yaml

```diff
@@ -33,16 +33,9 @@ on:
         default: false
       commands:
```

Comment:
We need to rename this input variable now that it accepts a single command instead of a string-ified list